### PR TITLE
184461071 - Update document addressing CVEs, stemcells and cflinuxfs

### PR DESCRIPTION
**Pull request by nimalank7**
What
----
- Updates documentation on how to address CVEs
- Updates documentation on how to upgrade stemcells and cflinuxfs
- Removes redundant information on CVEs from other document

How to review
-------------
Run locally to test changes

Who can review
--------------

@cmcnallygds 

Describe who can review the changes. Or more importantly, list the people
that can't review, because they worked on it.

### CODE REVIEW

File: source/guides/cve_stemcell_cflinuxfs.html.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## How to triage CVEs
+
+To triage CVEs, you should consider:
+
+- which libraries or tools are affected
+- how severely the libraries or tools are affected
+- how the vulnerability affects us or tenants
+- the likelihood of the vulnerability being exploited – for example, what kind of library or program is it? Would it be public facing in some way? Is it likely to be fed user input in a tenant application? What would an attacker need to do to get into a position where they could exploit it on PaaS?
+
+You should then record the priority decision in the ticket.
```
Comment (cadmiumcat) [inline source/guides/cve_stemcell_cflinuxfs.html.md:28]: Just to make it clear that we don't mean P1, P2... unless that's what we mean?
```suggestion
You should then record the priority decision (high or low) in the ticket.
```

**Review by cadmiumcat — COMMENTED** [review]
Looks good. I've added some minor suggestions

File: source/guides/cve_stemcell_cflinuxfs.html.md (hunk 1 above)
Comment (nimalank7) [reply source/guides/cve_stemcell_cflinuxfs.html.md:28]: Yep you're right that's much clearer.

File: source/guides/cve_stemcell_cflinuxfs.html.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Responding to CVEs, upgrading stemcells and cflinuxfs
+---
+
+# Responding to CVEs, upgrading stemcells and cflinuxfs
+
+This document explains how to triage common vulnerabilities and exposures (CVEs). 
+Most CVEs also require upgrading the [stemcells](https://bosh.io/docs/stemcell/) and [cflinuxfs](https://github.com/cloudfoundry/cflinuxfs3).
```
[the remark this answers is not in the document]
Comment (nimalank7) [reply source/guides/cve_stemcell_cflinuxfs.html.md:8]: I'm happy with that change - @cmcnallygds? In the [incident doc](https://team-manual.cloud.service.gov.uk/incident_management/incident_process/) it does use 

> This document is the GOV.UK PaaS team playbook for managing

which is were I originally copied it from.

[the remark this answers is not in the document]
Comment (cadmiumcat) [reply source/guides/cve_stemcell_cflinuxfs.html.md:8]: That's fair, i hadn't spotted that. Feel free to ignore this 👍

**Review by cadmiumcat — APPROVED** [review]

File: source/guides/cve_stemcell_cflinuxfs.html.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nimalank7) [reply source/guides/cve_stemcell_cflinuxfs.html.md:8]: @cmcnallygds also agrees with you so I'll make the change!

**Review by cadmiumcat — APPROVED** [review]

**Pull request closed without merging**